### ISSUE TITLE
Unable to parse correctly on volumes larger than 5Gb

### ISSUE THREAD

**Issue by nblog** [Forensics-Lab/ReadFS#4]
when my volume length is greater than or equal to 5Gb, it leads to a `container table` exception, and since I don't know much about the ReFS filesystem, I can only guess if it's because of multiple `Ministores` nested in the `container table`?

**Comment by nblog** [comment]
When my ReFS volume size is 6GB, the first time the `Container Table` is parsed in `setContainerTableEntries`, it's no longer [Container Table Structural](https://github.com/Forensics-Lab/ReadFS/blob/3b77324fa4ea038f43e14c58c1c2381c5785e7c0/ReFS/Index/indexEntries.py#L56), it's been inserted into a new `Node` for storage

6GB Volume
![6GB Volume](https://github.com/Forensics-Lab/ReadFS/assets/10218627/b62d6e16-c433-47a5-8967-bc3cfa0fefdb)


5GB Volume
![5GB Volume](https://github.com/Forensics-Lab/ReadFS/assets/10218627/12602f9a-b413-4706-8682-d01e040d4347)

**Comment by 1Doomdie1** [comment]
Hmmm, interesting discovery. I've also tried with volumes that are 10GB and everything worked fine. 
![image](https://github.com/Forensics-Lab/ReadFS/assets/43732125/497c9815-e6f1-4ea4-8bed-4956ab1d91ee)

**Comment by nblog** [comment]
I think the internal class should have a `reader` and the parameter being limited to `bytes` would interrupt reading the data again. So I can't begin to fix the above problem.

Here I describe below my analyzing environment:
`Windows 10 Enterprise LTSC 2021 x64`, which itself has ReFS 3.4 enabled

**Comment by 1Doomdie1** [comment]
I don't think that the Windows version matters too much, although I will take a look and come back to you with more info.

**Comment by nblog** [comment]
@1Doomdie1 
I'm sorry, I didn't read your message clearly, we're different and it's causing problems!
Allocation unit sizes are not the same，when I select `64K` as well, it works fine
![image](https://github.com/Forensics-Lab/ReadFS/assets/10218627/c812f1e2-cace-4929-9236-62e5d312a6a4)

My output message:
![image](https://github.com/Forensics-Lab/ReadFS/assets/10218627/c9137704-9aab-4ac8-af80-e0bb94ac8fdb)

**Comment by 1Doomdie1** [comment]
Ah yes, I haven't implemented a stable version for volumes that have the cluster size smaller than 64kb. I'll have to work on that. Totally sliped my mind, sorry.
